website/docs/r/is_reservation.html.markdown

```diff
@@ -0,0 +1,172 @@
+---
+
+subcategory: "VPC infrastructure"
+layout: "ibm"
+page_title: "IBM : reservation"
+description: |-
+  Manages IBM reservation.
+---
+
+# ibm_is_reservation
+Create, update, or delete a reservation.
+
+**Note:** 
+VPC infrastructure services are a regional specific based endpoint, by default targets to `us-south`. Please make sure to target right region in the provider block as shown in the `provider.tf` file, if VPC service is created in region other than `us-south`.
```

Comment:
link please

Reply:
There is no link yet, I will update once links are available